packages/shared-ux/page/no_data/impl/src/no_data_page.tsx

```diff
@@ -33,13 +33,13 @@ export const NoDataPage = ({
       values: { solution },
     });
 
-  const link = (
+  const link = docsLink ? (
```

Comment:
Making `docsLink` here optional to skip the docs link for serverless search empty page for now